hacktoberfest-2020.md

```diff
@@ -0,0 +1,55 @@
+## Hacktoberfest 2020 and Tekton!
+
+### What's this about?
+
+We'd love to encourage folks to contribute to Tekton - and over at the [Continuous Delivery Foundation](https://cd.foundation/) we've decided something like this could come in handy.
+
+### Guidance for project maintainers
+
+When creating a new issue for Hacktoberfest, please assign it the Hacktoberfest-2020 label and make sure you provide at least:
+
+- A perceived difficulty level
+- Awesome contacts/mentors willing to help out
+- An idea of what we'd like to achieve
+- Any useful pointers for getting started
+
+### Guidance for contributors
+
+The below links all contain the label selector for hacktoberfest-2020 and there you can find all of the issues that the maintainers think would be good ones to get involved in
+
```

Comment:
```suggestion
In case of a doubt, don't hesitate to contact the issue creator or the project maintainers to get things cleared out

If you start addressing an issue please assign yourself to it by posting the `/assign` message in the issue, it will help maintainers know who is working on it and let others know that the issue has already been picked up by someone
```

How about adding something like this ?

Reply:
Done, thanks for the suggestion